scripts/run_performance_job.py

```diff
@@ -531,16 +531,25 @@ def run_performance_job(args: RunPerformanceJobArgs):
 
     runtime_type = ""
 
+    # dotnet/runtime AndroidSampleApp using Mono runtime
     if android_mono:
         runtime_type = "Mono"
         configurations["CompilationMode"] = "JIT"
         configurations["RuntimeType"] = str(runtime_type)
 
+    # dotnet/runtime AndroidSampleApp using CoreCLR runtime
     if android_coreclr:
         runtime_type = "CoreCLR"
         configurations["CompilationMode"] = "JIT"
         configurations["RuntimeType"] = str(runtime_type)
 
+    # .NET Android and .NET MAUI Android sample app scenarios
+    if args.run_kind == "maui_scenarios_android":
+        if not args.runtime_flavor in ("mono", "coreclr"):
+            raise Exception("Runtime flavor must be specified for maui_scenarios_android")
+        configurations["CodegenType"] = str(args.codegen_type)
```

Comment:
Just for my understanding how is `CodegenType` different from `CompilationMode` used above?

Reply:
Ok, I was referring to lines:
https://github.com/dotnet/performance/pull/4770/files#diff-75e8f68d6caa7704e8c72407bcfc108e368c9b3ea7e71ca1532033c8905bad46R543
https://github.com/dotnet/performance/pull/4770/files#diff-75e8f68d6caa7704e8c72407bcfc108e368c9b3ea7e71ca1532033c8905bad46R537

Reply:
I think these are old version, `CompilationMode` is no longer being used there.